internal/injector/config/schema.json

```diff
@@ -881,8 +881,22 @@
           "prepend-statements": {
             "title": "Add new logic before a node",
             "markdownDescription": "The `prepend-statements` advice inserts new statements rendered by the provided code template before the matched AST node. This is often used to add logic in the preamble of function implementations, and the `defer` keyword can be used to also introduce epilogue logic.",
-            "$ref": "#/$defs/code-template",
-            "unevaluatedProperties": false
+            "unevaluatedProperties": false,
+            "allOf": [
+              { "$ref": "#/$defs/code-template" },
+              {
+                "properties": {
+                  "order": {
+                    "description": "The execution order within the namespace. Lower values execute first.",
+                    "type": "integer"
+                  },
+                  "namespace": {
+                    "description": "The logical grouping namespace for this advice.",
+                    "type": "string"
+                  }
```

Comment:
So it is legal to specify `order` but not `namespace` (and vice-versa)?

I'm cool with this if intentional... it might be easier to reason over consequences of setting it if one implies the other is required? (Similar to the comment above on the selection of the `DefaultNamespace` value; which BTW should probably be mentioned here for documentation purposes).